### PR TITLE
Cold start automation releated changes

### PR DESCRIPTION
**Pull request by shhsu**
Included some fixes and extension of existing scripts and also added IIS scripts

### CODE REVIEW

**Comment by shhsu** [comment]
@sivagms  @SajayAntony

File: test/ColdStart/Archive-IIS.ps1 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+param (
+    [Alias("t")]$targetApp = "HelloWorldMvc",
+    [Alias("f")]$framework = "netcoreapp1.0"
+)
+
+$coldSiteName = "${targetApp}Cold"
```
Comment (SajayAntony) [inline test/ColdStart/Archive-IIS.ps1]: Need a header or documentation on what this script does.

File: test/ColdStart/Archive.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```
Comment (SajayAntony) [inline test/ColdStart/Archive.sh]: Doc please. Does this create a CSV and move stuff etc.?

File: test/ColdStart/Archive.ps1 [hunk 3]
```diff
@@ -0,0 +1,42 @@
+param (
+    [Alias("t")]$targetApp = "HelloWorldMvc",
+    [Alias("f")]$framework = "netcoreapp1.0"
+)
+
```
Comment (SajayAntony) [inline test/ColdStart/Archive.ps1]: Script purpose?

Comment (shhsu) [inline test/ColdStart/Archive.ps1]: These aliases are for script purpose and friendly usage.

**Comment by shhsu** [comment]
Added Docs

File: test/ColdStart/Measure-nginx.sh [hunk 4]
```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+## This script perform 1 iteration of cold start measurements
```
Comment (SajayAntony) [inline test/ColdStart/Measure-nginx.sh]: perform`s`, write`s`

Comment (SajayAntony) [inline test/ColdStart/Measure-nginx.sh]: Might have to fix this in other places as well. 

---
In reply to: [68280650](https://github.com/aspnet/Performance/pull/154#discussion_r68280650) [](ancestors = 68280650)

**Comment by SajayAntony** [comment]
:shipit:

**Pull request merged**